src/api/app/services/meta_controller_service/project_updater.rb

```diff
@@ -0,0 +1,30 @@
+module MetaControllerService
+  class ProjectUpdater
+    def initialize(project: nil, request_data: {}, validator_klass: ::MetaControllerService::RequestValidator)
+      @project = project
+      @request_data = request_data
+      @validator = validator_klass.new(project: project, request_data: request_data)
+    end
+
+    def call
```

Comment:
[nitpick] I would prefer to use ``save`` here to keep same interface as ActiveRecord.